### DISCUSSION TITLE
Used the Jetson Nano for the biped/Help

### DISCUSSION THREAD

**Discussion by titoletito** [makerforgetech/modular-biped#86] [Q&A]
Sorry to take up your time. I know that some of you have managed to use the Jetson Nano for various projects. I was wondering if anyone could come forward to explain this to a novice like me. Your help would be greatly appreciated if you have the patience to do so.

There are several aspects I’d like to explore, including all the possibilities that the Jetson Nano can offer and how to use them properly. When it comes to printing, connecting motors, and peripherals, I think I can manage. However, I’m concerned that coding might be my main challenge.
The Jetson is arriving in a week, so I’m preparing with your help. If you have any tips or insights, please feel free to share. I’m really far from understanding all the parameters! :p

Thank you for taking the time to read this. :)

have good day :)

**Comment by danic85** [comment]
Hi @titoletito! Yes we have a few people looking at the nano for this project. If you're looking for general advice you might find more support on the discord as things are more active over there: https://bit.ly/makerforge-community

**Reply by titoletito** [reply]
Thanks you bro :p
